src/components/_product/Parameters/Parameters.module.scss

```diff
@@ -0,0 +1,44 @@
+.Parameters {
+  margin: clamp(0px, calc(16vw/7.68), 28px);
+
+  @media (max-width: 480px) {
+    margin: 0px;
+  }
```

Comment:
I would rather declare only `min-width`. So:
```
@media (min-width: 480px) {
  margin: clamp(0px, calc(16vw/7.68), 28px);
}
```

Reply:
Some type of mobile screens can have problems with float px width, for example screen can be 840.2px wide and it will cause layout shift. Preferable to not use max and min width media query on the same page.